modules/drivers/ili9341/modIli9341.c

```diff
@@ -205,6 +215,7 @@ void xs_ILI9341(xsMachine *the)
 
 	modSPIConfig(sd->spiConfig, MODDEF_ILI9341_HZ, MODDEF_ILI9341_SPI_PORT,
```

Comment:
`modSPIConfig` needs to initialize the `mode` field to the current default, otherwise all calling code would need to be updated to set the mode (which is impractical).